api/.env.test

```diff
@@ -6,3 +6,4 @@ PANTHER_APP_ENV=panther
 PANTHER_ERROR_SCREENSHOT_DIR=./var/error-screenshots
 API_DOMAIN=example.com
 COOKIE_PREFIX=example_com_
+TRANSLATE_ERRORS_TO_LOCALES="en,en_CH_scout,de,de_CH_scout,fr,fr_CH_scout,it,it_CH_scout"
```

Comment:
The syntax with underscores was invented for Java I think. The syntax with dashes is the [IETF language tag format](https://en.wikipedia.org/wiki/IETF_language_tag). On the web, the IETF dash syntax is more common (e.g. Accept-Language header, html lang="" attribute). Do you need to use underscores here, or could we use dashes as well?

Reply:
I tested it quickly, api platform does not support dashes...